.chloggen/default_telemetry_settings.yaml

```diff
@@ -0,0 +1,19 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
+
+# The name of the component, or a single word describing the area of concern, (e.g. collector, target allocator, auto-instrumentation, opamp, github action)
+component: collector
+
+# A brief description of the change. Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Expose the Collector telemetry endpoint by default.
+
+# One or more tracking issues related to the change
+issues: [3361]
+
+# (Optional) One or more lines of additional information to render under the primary note.
+# These lines will be padded with 2 spaces and then inserted directly into the document.
+# Use pipe (|) for multiline entries.
+subtext: |
+  In the upcoming collector version v0.111.0 the telemetry metrics endpoint will by default switch to localhost.
```

Comment:
The collector 0.111.0 changes the default binding of the telemetry metrics endpoint from `0.0.0.0` to `localhost`.

Reply:
```suggestion
  The collector v0.111.0 changes the default binding of the telemetry metrics endpoint from `0.0.0.0` to `localhost`.
```